extern/qgv/qgv/QGVCore/private/QGVCore.h

```diff
@@ -93,7 +93,9 @@ class QGVCore
         rdr.len = strlen(cp);
         rdr.cur = 0;
 
+#ifndef GRAPHVIZ_VERSION_GE_9
         disc.mem = &AgMemDisc;
```

Comment:
Unclear: which are the implications of removing `disc.mem`? Is it safe?

Reply:
In graphviz, there is not anymore any support for custom memory allocators, see https://gitlab.com/graphviz/graphviz/-/blob/main/CHANGELOG.md#900-2023-09-11  :  
>  Breaking: The concept of “memory allocator discipline” has been removed,
>  along with the type Agmemdisc_t and fields Agdisc_t.mem and
>  Agdstate_t.mem.

The default memory allocators are used by default in graphviz 9.